src/test/java/app/bpartners/api/unit/mapper/UserRestMapperTest.java

```diff
@@ -49,31 +88,6 @@ void user_subscription_mapped_with_subscription_values() {
     assertEquals(expectedEndDatetime, actualSubscription.getEnd());
   }
 
-  @Test
-  void user_subscription_mapped_with_trial() {
```

Comment:
So from now, isn't there any test to check the "CANCELLED" subscription status ?